components/cloud/utils/states.ts

```diff
@@ -0,0 +1,79 @@
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export interface StateCode {
+    code: string;
+    name: string;
+}
+
+export const US_STATES = [
```

Comment:
There was no folder for `constants` in utils, so I moved the `states.ts` file into `utils` folder. Couldn't put in utils/constants.jsx because typescript and